pkg/codegen/testing/test/sdk_driver.go

```diff
@@ -266,6 +266,12 @@ var PulumiPulumiSDKTests = []*SDKTest{
 		Description: "Regress pulumi/pulumi#12971 affecting Go",
 		Skip:        allLanguages.Except("go/any"),
 	},
+	{
+		Directory:        "regress-go-15478",
+		Description:      "Regress pulumi/pulumi#15478 affecting Go ensuring SDK-gen doesn't panic",
+		Skip:             allLanguages.Except("go/any"),
+		SkipCompileCheck: codegen.NewStringSet(golang),
```

Comment:
Should we add a comment here? We're deliberately skipping right now because we don't expect it to be able to compile due to missing deeply nested container types in the core Go SDK.